onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -53,8 +60,9 @@ void ModelBuilder::AddInitializerToSkip(const std::string& tensor_name) {
 }
 
 Status ModelBuilder::Prepare() {
-  RETURN_STATUS_ON_ERROR(nnapi_->ANeuralNetworksModel_create(&nnapi_model_->model_));
-  ORT_RETURN_IF_ERROR(GetTargetDevices());
+  RETURN_STATUS_ON_ERROR(nnapi_.ANeuralNetworksModel_create(&nnapi_model_->model_));
+  // uncomment the following line to set the execution preference to [low power, fast single answer, low latency]
```

Comment:
what does "fast single answer" mean? - Zhijiang.

Reply:
It represents a ENUM value. fixed.